test/nft.test.js

```diff
@@ -8,15 +8,15 @@ import {
     alice,
     before,
     cleanup,
-    delay,
+    delay, estimatorAdmin,
     ExampleNFT,
     exampleNFTAdmin,
-    exampleTokenAdmin, getEventFromTransaction,
+    exampleTokenAdmin, getAccountBalances, getEventFromTransaction,
     lostAndFoundAdmin
 } from "./common";
 
 // Increase timeout if your tests failing due to timeout
-jest.setTimeout(10000);
+jest.setTimeout(100000);
```

Comment:
is this needed for every test? I feel like we should scope it on a per test basis ideally unless it's affecting everything. this is a long timeout

Reply:
From what I can tell, each test is taking ~6 seconds to execute. We can bring it down or we could probably move to overflow instead of flow-js-testing which I have anecdotally noticed is faster